k8s/src/main/scala/io/buoyant/k8s/README.md

```diff
@@ -0,0 +1,2 @@
+This package contains representations of objects returned by multiple versions of the Kubernetes API.
+Version-specific objects should go in sub-packages (see v1.scala).
```

Comment:
This file feels a bit buried to me -- we haven't previously included readme files nested this deeply within the project.  Maybe this information would be more visible as a scaladoc comment in k8s/src/main/scala/io/buoyant/k8s/Api.scala?